Jenkinsfile

```diff
@@ -36,12 +31,7 @@ pipeline {
     stage('Checkout') {
       options { skipDefaultCheckout() }
       steps {
-        //TODO we need to configure the library in Jenkins to use privileged methods.
-        //gitCheckout(basedir: "${BASE_DIR}")
-        dir("${BASE_DIR}"){
-          checkout scm
-          githubEnv()
-        }
+        gitCheckout(basedir: "${BASE_DIR}")
```

Comment:
this triggers the CI first time contributors validation, do we need it?

Reply:
PRs are disabled